lib/koans/07_pattern_matching.ex

```diff
@@ -44,12 +49,18 @@ defmodule PatternMatching do
     assert items == ___
   end
 
-  koan "Patterns show what you really care about" do
+  koan "Maps support partial pattern matching" do
     %{make: make} = %{type: "car", year: 2016, make: "Honda", color: "black"}
 
     assert make == ___
   end
 
+  koan "Lists must match exactly" do
```

Comment:
It seemed nice to draw the contrast between how matching works with maps vs. lists.